section0/0.4.md

```diff
@@ -0,0 +1,17 @@
+# ![Github](/images/github-big.png) Setting up GitHub
+
+----
+
+Git is a version control system used by most developers. We will setup Git in a moment. Before doing so we will first setup GitHub. 
+
+GitHub is a website for storing git repositories (repos). These are programming source files, and other assets. Git allows very powerful history tracking of your projects including the ability to revert. GitHub is a remote website for storing source files that are under git version control. This allows others to access, view, clone (download), alter and collaborate.
```

Comment:
... including the ability to revert changes (maybe better with the word changes)